_posts/2024-09-28-unstable-release-0.10.9.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: Stable release 0.10.9
+author: Saugia
+---
+
+If you like more Endless Sky, boy do I have the thing for you! v0.10.9 is now live on Github and Steam! As usual, GoG will be within a few days pending some QA testing. This is a hefty unstable release that brings all sorts of content additions, adjustments, balance changes, bug fixes, and so on! If you're itching to know everything that has changed, head on over to the changelog for the release, which can be found here:
+
+Here are some highlights coming with this release:
+
+When it comes to boarding and capturing, there never really has been a reason to incorporate defense as the game always calculates using the attack values of the AI versus your crew. With this update, if both sides choose to attack, two rounds of combat are run - one with the player's attack against the enemy's defense, and one with the enemy's attack against the player's defense, instead of using the player's attack against the enemy's attack. Now those thousands of Pug Biodefenses you farmed can be useful against any trigger-happy Pirate! Thanks to the efforts of Peter and Hurleveur. But that's not all that these two have for us!
```

Comment:
```suggestion
When it comes to boarding and capturing, there never really has been a reason to incorporate defense, as the game always calculates using the attack values of the AI versus your crew. With this update, if both sides choose to attack, two rounds of combat are run - one with the player's attack against the enemy's defense, and one with the enemy's attack against the player's defense, instead of using the player's attack against the enemy's attack. Now those thousands of Pug Biodefenses you farmed can be useful against any trigger-happy pirate, thanks to the efforts of Peter and Hurleveur. But that's not all that these two have for us!
```